pkgs/applications/science/biology/exonerate/default.nix

```diff
@@ -0,0 +1,23 @@
+{ stdenv, fetchurl, glib, pkgconfig }:
+
+stdenv.mkDerivation rec {
+  version = "2.4.0";
+  name = "exonerate-${version}";
+
+  src = fetchurl {
+    url = "http://ftp.ebi.ac.uk/pub/software/vertebrategenomics/exonerate/${name}.tar.gz";
+    sha256 = "0hj0m9xygiqsdxvbg79wq579kbrx1mdrabi2bzqz2zn9qwfjcjgq";
+  };
+
+  doCheck = true;
+
+  buildInputs = [ glib pkgconfig ];
```

Comment:
Programs executed during build should go to `nativeBuildInputs`.

Reply:
Yes, I fixed that, thanks!